packages/accounts-2fa/README.md

```diff
@@ -0,0 +1,172 @@
+#accounts-2fa
+
+---
+
+Easy 2-Factor Integration For Meteor Apps
```

Comment:
Easy is not the best word here.

Simple would be better but in this case I would say:
`2-Factor Authentication for Meteor apps using accounts-password`